CMakeLists.txt

```diff
@@ -15,6 +15,13 @@ endif()
 project(SwiftTesting
   LANGUAGES CXX Swift)
 
+if(NOT APPLE)
+  if(NOT CMAKE_SYSTEM_NAME STREQUAL WASI)
+    find_package(dispatch CONFIG)
+  endif()
```

Comment:
Any reason to not just do this unconditionally? We could do this, it shouldn't prevent the configure or build.

Reply:
Just to make it clear `WASI` won't use dispatch. Although we can do this unconditionally, I thought there's no reason we'd call this for WASI.

Reply:
Right, I get that. My reasoning is that we should eventually get dispatch working on WASI as well - we want the platforms to be at parity whenever possible.